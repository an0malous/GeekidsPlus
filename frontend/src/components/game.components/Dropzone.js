import React from 'react';

export const Dropzone = (props) => {
  
    return (
        <div style={{height: "210px"}}className="ui grid center aligned container">
            {props.dropzoneWord.map(zone=><div style={{width: "200px", height: "200px", background: "black"}}className="dropzone inner-dropzone" key={zone}>{zone}</div>)}
        </div>
         
       
    )
}