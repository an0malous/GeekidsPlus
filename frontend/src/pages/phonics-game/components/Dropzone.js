import React from 'react';

export const Dropzone = (props) => {
  {console.log(props.dropzoneWord)}
    return (
        <div style={{height: "210px"}}className="ui grid center aligned container">
            {props.dropzoneWord.map(zone=><div style={{width: "200px", height: "200px", background: "black"}} className="dropzone inner-dropzone" key={zone}>{zone}</div>)}
        </div>
         
       
    )
}