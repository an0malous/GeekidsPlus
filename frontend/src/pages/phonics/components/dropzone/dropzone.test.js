import React from 'react'
import renderer from 'react-test-renderer'
import Dropzone from './dropzone.component';

describe('Dropzoneu Component', () => {
   let component;

   beforeEach(() => {
     component = renderer.create(<Dropzone />).toJSON()
   });
   
   it('Renders', () => {
      expect(component).toMatchSnapshot()
   })
});

