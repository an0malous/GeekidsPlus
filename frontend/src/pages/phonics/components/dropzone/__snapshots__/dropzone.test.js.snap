// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropzone Component Renders without crashing 1`] = `<div />`;
